### ISSUE TITLE
[BUG] QColumnView Size Grip Expands View

### ISSUE THREAD

**Issue by Alexhuszagh** [Alexhuszagh/BreezeStyleSheets#54]
## Description

Hovering over the size grips in a QColumnView expands the widget, and does not trigger the size grips correctly. Dissecting the issue, I've found the offending commit is the patch for #11, specifically, d620f2226b574b560e3041f5daf27b68d9aca620, and the offending code is:

```css
QWidget:disabled:hover
{
    background-color: ^background^;
}
```

Removing this fixes the issue, however, it might cause ripple effects.
